README.md

```diff
@@ -1,4 +1,23 @@
-# zombie-game-workshops
+# Gra zombie - warsztaty z JavaScript i Vue
+
+Cześć, napiszemy dzisiaj razem grę! 🧟️
+
+## Jak zacząć?
+Musisz zainstalować: 
+- Git (https://git-scm.com/downloads)
+- Node (https://nodejs.org/en/)
+- VSCode (https://code.visualstudio.com/) albo inny edytor kodu
+
+Otwórz terminal (PowerShell na Windows), skopiuj, wklej i wykonaj poniższe komendy:
+
+```
+git clone git@github.com:hatimeria/zombie-game-workshops.git
+cd zombie-game-workshops
+npm install
+npm run serve
+```
+
+I otwórz http://localhost:8080/ w przeglądarce ✨
 
 ## Project setup
 ```
```

Comment:
Jak zainstalują Gita i Node'a to bym zrestartowała komputer 😄 To powinno wszystko ułatwić 😁

Reply:
Hmm, może jeśli będą to robić na Windowsie, to i tak dostaną komunikat, że muszą? Aaa, ja bym im w ogóle powiedziała, żeby gita i node'a zainstalować przed zajęciami? Rozesłać taką prośbę wcześniej w mailu do uczestniczek? Tutaj takie przypomnienie. Ale w sumie to są też pewnie szczegóły do ustalenia potem?